index.html

```diff
@@ -155,7 +155,8 @@ <h3>{{RTCPeerConnection/addTransceiver()}}</h3>
            {{RTCRtpTransceiverInit/sendEncodings}} within
            {{RTCPeerConnection/addTransceiver()}}. To validate
            {{RTCRtpEncodingParameters/scalabilityMode}}, add
-           the following validation steps after step 3 (within step 8):
+           the following steps to the
+           [=RTCPeerConnection/addTransceiver sendEncodings validation steps=]:
```

Comment:
Unless dropping position here was intentional, we should keep it since the observable order of errors matter for testing and web compat.

Also maybe s/add/insert/ ? (github won't let me suggest across deleted lines)
```suggestion
           the following steps after step 3 of the
           [=RTCPeerConnection/addTransceiver sendEncodings validation steps=]:
```

Reply:
The same error type is returned regardless of which check fails first. The error strings are UA dependent and may or may not have any information about what is happening, so they are not part of the specified API and should not be relied upon nor be observed.